moveit_setup_assistant/src/widgets/planning_groups_widget.cpp

```diff
@@ -102,10 +102,12 @@ PlanningGroupsWidget::PlanningGroupsWidget(QWidget* parent, moveit_setup_assista
   QVBoxLayout* layout = new QVBoxLayout();
 
   // Top Label Area ------------------------------------------------
-  HeaderWidget* header =
-      new HeaderWidget("Planning Groups", "Create and edit planning groups for your robot based on joint collections, "
-                                          "link collections, kinematic chains and subgroups.",
-                       this);
+  HeaderWidget* header = new HeaderWidget(
+      "Planning Groups", "Create and edit planning groups for your robot based on joint collections, "
+                         "link collections, kinematic chains and subgroups.\n"
```

Comment:
Please change this `and` to *or*, or make it otherwise clear that not everything is required.
There were complains about this being unclear before.

Reply:
Good point.